core/domain/exp_domain.py

```diff
@@ -3950,11 +3954,11 @@ def to_metadata_dict(self):
         id, title and objective of the exploration.
 
         Returns:
-            A metadata dict for the given exploration summary.
+            dict. A metadata dict for the given exploration summary.
             The metadata dict has three keys:
-                - 'id': str. The exploration ID.
-                - 'title': str. The exploration title.
-                - 'objective': str. The exploration objective.
+                - 'id': (str). The exploration ID.
```

Comment:
Please revert this to what it was before, don't use parens. Ditto for the next two args.